packages/runtime/test/integration/global-opcodebudget.ts

```diff
@@ -0,0 +1,141 @@
+import { parsing, types } from "@algo-builder/web";
+import { getApplicationAddress } from "algosdk";
+import { assert } from "chai";
+
+import { AccountStore, getProgram, Runtime } from "../../src/index";
+import { LogicSigAccount } from "../../src/logicsig";
+import { AppDeploymentFlags } from "../../src/types";
+import { useFixture } from "../helpers/integration";
+
+describe("TEALv6: Global Opcode Budget", function () {
+	useFixture("global-opcodebudget");
+	const john = new AccountStore(10e6);
+
+	let runtime: Runtime;
+	let flags: AppDeploymentFlags;
+	let appID: number;
+	let dummyAppID: number;
+	let txnParam: types.ExecParams;
+	let lsig: LogicSigAccount;
+	this.beforeAll(async function () {
+		runtime = new Runtime([john]); // setup test
+
+		flags = {
+			sender: john.account,
```

Comment:
indentation is wrong. Could you please check your editor? A tab should be set to 2 spaces for javascript.

Reply:
I remember we are ran prettier in husky hook before commit. I will check it.